.gitlab-ci.yml

```diff
@@ -50,7 +50,7 @@ DeepSet:new_samples_baseline_5param_extended_trk:
     CMSSW_VERSION: 'CMSSW_15_1_0_pre1'
     CMSSW_L1CT: 'CMS-L1T-Jet-Tagging:for-DPS'
     CMSSW_EMULATOR_WRAPPER: 'https://github.com/cms-hls4ml/L1TSC4NGJetModel.git'
-    RERUN_ON_TAG: 'False'
+    RERUN_ON_TAG: 'True'
```

Comment:
What was the reason this being changed to true, do we want the pipeline to run on a newly trained model or the tagged model?

Reply:
Ah this was just because this PR didn't affect the model at all, just the evaluation so I was trying to bypass the unnecessary retraining in CI to see results.

But I should change it back before we merge